### PR TITLE
#1556: activate build profile avadoc-jdk-8u121 by default

### LINKED ISSUE #1556: Activate profile javadoc-jdk-8u121 by default
We should activate the `javadoc-jdk-8u121` in our maven build by default. The reasoning behind it is that more people are now using a new JDK 8 than an old one.

### PR DESCRIPTION
**Pull request by andrii-kovalenko-ct**
resolves #1556

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -547,11 +547,8 @@
         </profile>
         <profile>
             <id>javadoc-jdk-8u121</id>
-            <!--
```
Comment (katmatt) [inline pom.xml]: Great that you removed this now unnecessary comment :1st_place_medal:

**Comment by andrii-kovalenko-ct** [comment]
@katmatt 
>Only change request: Can you please remove the profile from the .travis file?

6e27484013

**Comment by katmatt** [comment]
@andrii-kovalenko-ct I'm really sorry, but looks like we have to keep the profile in the `.travis` file because we added another one recently. And as soon as you specify one, the default ones are disabled. So please revert your last commit and then we can merge your PR.

**Comment by andrii-kovalenko-ct** [comment]
@katmatt 
i had such a doubt, but i thought it is just a doubt :)
d764d8d5fd

**Comment by andrii-kovalenko-ct** [comment]
don't you think we should activate both `javadoc-jdk-8u121` and `dependency-check` by default?

**Comment by katmatt** [comment]
@andrii-kovalenko-ct No, `dependency-check` performs a security check of our dependencies and decreases the local build time a lot.

**Comment by andrii-kovalenko-ct** [comment]
ok, got it

**Pull request merged**